dali/driver/atxled.py

```diff
@@ -0,0 +1,190 @@
+from dali.command import Command
+from dali.driver.base import SyncDALIDriver, DALIDriver
+from dali.frame import ForwardFrame, BackwardFrame
+import logging
+import sys
+
+# avoid import name collision with 'serial' driver
+original_sys_path = list(sys.path)
+sys.path = [p for p in sys.path if not p.endswith("python-dali/dali/driver")]
+import serial
+sys.path = original_sys_path
+
+import threading
+import time
+
+DALI_PACKET_SIZE = {"j": 8, "h": 16, "l": 24, "m": 25}
+DALI_PACKET_PREFIX = {v: k for k, v in DALI_PACKET_SIZE.items()}
+
+
+class DaliHatSerialDriver(DALIDriver):
+    """Driver for communicating with DALI devices over a serial connection."""
+
+    def __init__(self, port="/dev/ttyS0", LOG=None):
+        """Initialize the serial connection to the DALI interface."""
+        self.port = port
+        self.lock = threading.RLock()
+        self.buffer = []
+        if not LOG:
+            self.LOG = logging.getLogger("AtxLedDaliDriver")
+            handler = logging.StreamHandler(sys.stdout)
+            self.LOG.addHandler(handler)
```

Comment:
If LOG is passed, you don't appear to be doing anything with it?

Reply:
Thanks! It will set self.LOG to the passed LOG now